data/templates/default.typst

```diff
@@ -92,7 +92,9 @@ $if(papersize)$
   paper: "$papersize$",
 $endif$
 $if(mainfont)$
-  font: ($for(mainfont)$"$mainfont$"$sep$, $endfor$),
+  font: ("$mainfont$"$if(mainfontfallback)$, $for(mainfontfallback)$"$mainfontfallback$"$sep$, $endfor$$endif$),
+$elseif(mainfontfallback)$
+  font: ($for(mainfontfallback)$"$mainfontfallback$"$sep$, $endfor$),
```

Comment:
I don't think we need to worry about a case where you have mainfontfallback but not mainfont -- the former only makes sense with the latter.